distribution/docker/src/docker/Dockerfile.wolfi

```diff
@@ -0,0 +1,177 @@
+################################################################################
+# This Dockerfile was generated from the template at distribution/src/docker/Dockerfile
+#
+# Beginning of multi stage Dockerfile
+################################################################################
+
+################################################################################
+# Build stage 1 `builder`:
+# Extract Elasticsearch artifact
+################################################################################
+
+FROM docker.elastic.co/wolfi/chainguard-base:latest AS builder
```

Comment:
My understanding is that in addition to `latest` there are other tags available. It's not clear to me which ones we should be using. I'd prefer latest if that's ok, so we don't have to keep updating this when there's a new relese, even though I believe eng prod has some tooling to automatically open/merge PRs for this sort of thing. We'll want to confirm which tag(s) we should use either way.